package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gw2e-scraping",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
Please don't increase the version manually, that is done by a script

Reply:
Corrected with latest commit